hyperspace/cosmos/src/lib.rs

```diff
@@ -68,6 +73,8 @@ pub struct CosmosClient<H> {
 pub struct CosmosClientConfig {
 	/// Chain name
 	pub name: String,
+	/// peer id for fetching light blocks through rpc io
+	pub peer_id: tendermint::node::Id,
```

Comment:
The `peer_id` should not be required as a configuration field, as it can be obtained via the rpc address. Check [here](https://github.com/informalsystems/hermes/blob/027578f04f6b53faacca6fa2bdbb9018288ee7dc/crates/relayer/src/chain/cosmos.rs#L1835) please